wordpress-mobile/WordPress-iOS
# Removed AbstractPostListViewControllerSubclass.h.

Removes the private headers from `AbstractPostListViewController.h`.  This may not seem necessary at first, but I'm planning to migrate this class to Swift, and Swift doesn't have support for private headers.

To test:
Since no source code was modified (only headers and include directives), making sure the app builds is enough.

If you want to double check:
1. Run the unit tests
2. Make sure the list of posts and the list of pages for any blog works fine.

Needs review: @jleandroperez 

/cc @aerych fyi
